src/docs/development/tools/devtools/visual-debugging.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Visual debugging
+description: A guide to Flutter's visual debugging functionality.
+---
+
+The Flutter Inspector provides several options for visually debugging your app.
+These are the options available from the inspector within the DevTools.
+
+## Slow Animations
+
+When enabled, this option runs animations 5 times slower for easier visual
+inspection.
+This can be useful if you want to carefully observe and tweak an animation that
+doesn’t look quite right.
+
+This can also be set in code:
+
+```dart
+import 'package:flutter/scheduler.dart' 
+
+timeDilation = 5.0;
+```
+ 
+This will slow the animations by 5x.
+
+### See also
+
+* [Flutter documentation: timeDilation property](https://api.flutter.dev/flutter/scheduler/timeDilation.html) 
+
+### Example
```

Comment:
These are GIF files, not actual example code. I'd move the before and after GIF files up to where it talks about how to slow animations (and delete the "Example" header).

```suggestion
The following GIF files show before and after slowing an animation.
```

Reply:
Agreed, but I don't think it's important that they're GIFs, so going with "The following screen recordings show before and after slowing an animation" if that's okay

Reply:
Sure.